client/src/components/Sle/Sle.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import "./Sle.css";
+import "./Sidebar.css";
+import "./Sidebar.jsx";
+import Sidebar from "./Sidebar";
+
+
+
+function Sle(props) {
+    return (
+    
+    <div>
+    
+    <div class = "line">
+    </div>
+    
+    <div class = "sidebar">
+      <Sidebar
+          title= "Your Shifts"/>
+      <Sidebar
+          title= "Open Shifts"/>
+      <Sidebar
+          title= "Availability"/>
+    </div>
+
+    <div class = "top-right">
+      <div class="user-box">
+      <div class="user-id">
+      <div class="user-name"> Bianca Lee
```

Comment:
The user-name will change based on who is logged in. This is exactly what props are used for, so let's make this a prop rather than hardcoding Bianca